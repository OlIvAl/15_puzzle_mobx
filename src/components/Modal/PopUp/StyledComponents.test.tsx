import * as React from 'react';
import {
  StyledPopUpWrapper,
  StyledPopUpHeader,
  StyledPopUpContent,
  StyledPopUpCloseButton
} from './StyledComponents';
import renderer from 'react-test-renderer';

describe('PopUp StyledComponents', () => {
  it('StyledPopUpWrapper отображается', () => {
    const tree = renderer.create(<StyledPopUpWrapper />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('StyledPopUpHeader отображается', () => {
    const tree = renderer.create(<StyledPopUpHeader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
  it('StyledPopUpContent отображается', () => {
    const tree = renderer.create(<StyledPopUpContent />).toJSON();

    expect(tree).toMatchSnapshot();
  });
  it('StyledPopUpCloseButton отображается', () => {
    const tree = renderer.create(<StyledPopUpCloseButton />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});