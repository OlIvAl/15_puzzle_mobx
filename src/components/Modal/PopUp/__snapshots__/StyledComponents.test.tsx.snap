// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUp StyledComponents StyledPopUpCloseButton отображается 1`] = `
.c0 {
  border-radius: 4px;
  padding: 6px 10px;
  background-color: #358ada;
  color: #fff;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  width: 50%;
  margin-top: 20px;
}

<button
  className="c0"
/>
`;

exports[`PopUp StyledComponents StyledPopUpContent отображается 1`] = `
<div
  className=""
/>
`;

exports[`PopUp StyledComponents StyledPopUpHeader отображается 1`] = `
.c0 {
  font-size: 24px;
  font-weight: bold;
  margin-bottom: 12px;
}

<div
  className="c0"
/>
`;

exports[`PopUp StyledComponents StyledPopUpWrapper отображается 1`] = `
.c0 {
  width: 300px;
  padding: 20px;
  background-color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
